### PR TITLE
docs: list pre-pulled versions for Azure Stack

### PR DESCRIPTION
**Pull request by jadarsie**
**Reason for Change**:
Update to Azure Stack doc page. List pre-pulled images for private preview users.

### CODE REVIEW

File: docs/topics/azure-stack.md [hunk 1]
```diff
@@ -170,4 +187,8 @@ Because Azure and Azure Stack currently rely on a different version of the Compu
 
 This can be resolved by making a small modification to the extension `template.json` file. Replacing all usages of template parameter `apiVersionDeployments` by the hard-code value `2017-12-01` (or whatever API version Azure Stack runs at the time you try to deploy) should be all you need.
 
-Once your are done updating the extension template, host the extension directory in your own Github repository or storage account. Finally, at deployment time, make sure that your cluster definition points to the new [rootURL](https://github.com/Azure/aks-engine/blob/master/docs/topics/extensions.md#rooturl).
\ No newline at end of file
+Once your are done updating the extension template, host the extension directory in your own Github repository or storage account. Finally, at deployment time, make sure that your cluster definition points to the new [rootURL](https://github.com/Azure/aks-engine/blob/master/docs/topics/extensions.md#rooturl).
```
Comment (mboersma) [inline docs/topics/azure-stack.md]: Typo: **you** are done updating...

File: docs/topics/azure-stack.md [hunk 2]
```diff
@@ -170,4 +187,8 @@ Because Azure and Azure Stack currently rely on a different version of the Compu
...
+
+### Troubleshoting
+
+This [how-to guide](/docs/howto/troubleshooting.md) has a good high level explanation of how AKS Engine interacts with the Azure Resource Manager (ARM) and lists a few potential issues that can cause AKS Engine commands to fail. Checkout the scripts in this [repository](https://github.com/msazurestackworkloads/azurestack-gallery/tree/master/diagnosis) to simplify the logs collection task.
```
Comment (mboersma) [inline docs/topics/azure-stack.md]: "high-level" should be hyphenated.

File: docs/topics/azure-stack.md [hunk 3]
```diff
@@ -153,11 +158,23 @@ If you need to expose more than 5 services, then the recommendation is to route
...
+<!-- - 1.14.3 build and push to mcr -->
+<!-- - 1.13.7 build and push to mcr -->
+- 1.14.1
 - 1.13.5
 - 1.12.8
+- 1.12.7
 - 1.11.10
 - 1.11.9
 
+If the Azure Stack instance is air gapped, then the syndicated [AKS Base Image](#disconnected-azure-stack-instances) will determine the Kubernetes version that you can deploy or the target upgrade version. The table below lists the pre-pulled versions on each AKS Base Image version.
```
Comment (mboersma) [inline docs/topics/azure-stack.md]: "air-gapped" should be hyphenated.

File: docs/topics/azure-stack.md [hunk 4]
```diff
@@ -153,11 +158,23 @@ If you need to expose more than 5 services, then the recommendation is to route
 
 These are the Kubernetes versions that you can deploy to Azure Stack using AKS Engine:
 
+<!-- - 1.14.3 build and push to mcr -->
+<!-- - 1.13.7 build and push to mcr -->
+- 1.14.1
```
Comment (mboersma) [inline docs/topics/azure-stack.md]: I don't see `1.14.1-azs` in the VHD script. I can add it, assuming this is correct.

File: docs/topics/azure-stack.md (hunk 1 above)
Comment (jadarsie) [reply docs/topics/azure-stack.md]: thx!

File: docs/topics/azure-stack.md (hunk 4 above)
Comment (jadarsie) [reply docs/topics/azure-stack.md]: you are the best, thx.

**Pull request merged**